### PR TITLE
improve examples and comments

### PR DESCRIPTION
**Pull request by Emilgardis**

### CODE REVIEW

**Comment by Nerixyz** [comment]
Could you fix the indentation and spacing (after the `=`) of this line here as well:

https://github.com/twitch-rs/twitch_api/blob/38dca416db2707b85e35bb582c8495e0cc8a60c4/README.md?plain=1#L25

This could probably use the same `Create the HelixClient, which is used to make requests to the Twitch API` comment as in the examples.

File: src/lib.rs [hunk 1]
```diff
@@ -14,34 +14,57 @@
...
 //!
+//! Get a channel
+//!
+//! ```rust,no_run
+//! use twitch_api::helix::HelixClient;
+//! use twitch_api::twitch_oauth2::{AccessToken, UserToken};
+//!
+//! #[tokio::main]
+//! async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+//!     // Create the HelixClient, which is used to make requests to the Twitch API
```
Comment (Emilgardis) [inline src/lib.rs:25]: this comment made rustfmt not format this as

```rs
//! #[tokio::main]
//! async fn main(
//! ) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
```

strange

Comment (Emilgardis) [reply src/lib.rs:25]: ah, it breaks rustfmt, nice

Comment (Nerixyz) [reply src/lib.rs:25]: Maybe that's a rustfmt bug 🙃.

Comment (Emilgardis) [reply src/lib.rs:25]: probably https://github.com/rust-lang/rustfmt/issues/5627 due to https://github.com/twitch-rs/twitch_api/blob/38dca416db2707b85e35bb582c8495e0cc8a60c4/rustfmt.toml#L10

**Comment by Emilgardis** [comment]
bors cancel

**Pull request merged**